comictaggerlib/autotagmatchwindow.py

```diff
@@ -89,6 +86,9 @@ def __init__(
 
         self.update_data()
 
+    def fetch(self, match: IssueResult) -> GenericMetadata:
+        return self.current_talker().fetch_comic_data(issue_id=match.issue_id)
+
```

Comment:
It looks like `self.current_talker()` needs to be changed to `self.talker` and `self.talker = talker` in the `init`

Reply:
Also, it's not visible here but `save_match` L:230 `md, error = self.parent().read_selected_tags(self._tags, ca)` needs to be changed to `md, _, error = self.parent().read_selected_tags(self._tags, ca)`

I would guess the same is true for `renamewindow.py` at L:85